onnxruntime/contrib_ops/cpu/skip_layer_norm.cc

```diff
@@ -45,18 +45,40 @@ Status SkipLayerNorm<T>::Compute(OpKernelContext* p_ctx) const {
 
   const auto& input_dims = input->Shape().GetDims();
   size_t input_dims_size = input_dims.size();
+  const auto& skip_dims = skip->Shape().GetDims();
+  size_t skip_dims_size = skip_dims.size();
+  int skip_sequence_length = static_cast<int>(skip_dims[skip_dims_size - 2]);
```

Comment:
Need to check skip_dims_size == 3 before this line. Otherwise skip_dims[skip_dims_size - 2] might throw exeception when skip_dims_size < 2.